Documents/idea document 2/ideadocument2.md

```diff
@@ -10,5 +10,16 @@
 # Speficications & Execution Plan
 
 # Expected Outcomes
+As mentioned earlier in the objective, the goal of this application is to identify and characterise neighborhoods in Amsterdam. In the following mockup of the user interface(UI) it will become visible how this could look.
+<p align="center">
+  <img alt="Workbench electron-browser component" src="../images/mockup.png" width = "70%">
+  <br>Figure: Mockup of the User Interface
+</p>
+In the UI there is a big map of Amsterdam. On this map blue dots are shown which are tweets or strava activities that are found to be relevant by the algorithm. Around those posts the neighborhood boxes are drawn in different colors to quickly make clear how the neighborhoods of Amsterdam are divided based on sport activities. Because only relevant tweets are shown this can give an indication to the user how active an area is with regards to sports.
```

Comment:
I thought feedback of Achilleas was that no single tweets should be presented? Or explain why this tweet is selected